ml-proto/spec/memory.ml

```diff
@@ -72,7 +72,7 @@ let grow mem n =
   let old_size = size mem in
   let new_size = Int64.add old_size n in
```

Comment:
For consistency with `create`, please change `grow` to also take a number of pages now.